### PR TITLE
CSI raw block to beta

### PR DESCRIPTION
**Pull request by bswartz**
Move CSI raw block support from alpha to beta

### CODE REVIEW

File: content/en/docs/concepts/storage/volumes.md [hunk 1]
```diff
@@ -1217,7 +1217,7 @@ persistent volume:
 
 #### CSI raw block volume support
 
-{{< feature-state for_k8s_version="v1.11" state="alpha" >}}
```
Comment (msau42) [inline content/en/docs/concepts/storage/volumes.md]: feature-gates.md should also be updated

**Comment by bswartz** [comment]
Is the the right patch to add a release note? Or are release notes handled though a different process?

**Comment by msau42** [comment]
Release notes are generated from the PR in k/k: https://github.com/kubernetes/kubernetes/pull/74909#issue-258098233

**Comment by jimangel** [comment]
@bswartz it appears this doc is ready to review, can you confirm? Also, can I get a technical lgtm?

**Comment by bswartz** [comment]
@jimangel Yes, it's a very small change because this particular feature gate is mostly aimed deployers and at storage vendors who create plugins. I haven't found any parts of the website documentation that really need to reflect this update other than the feature gates.

File: content/en/docs/concepts/storage/volumes.md (hunk 1 above)
Comment (msau42) [reply content/en/docs/concepts/storage/volumes.md]: The section below on the feature being turned off by default needs to be updated

Comment (bswartz) [reply content/en/docs/concepts/storage/volumes.md]: Done

File: content/en/docs/concepts/storage/volumes.md [hunk 2]
```diff
@@ -1217,19 +1217,20 @@ persistent volume:
...
 
 Starting with version 1.11, CSI introduced support for raw block volumes, which
 relies on the raw block volume feature that was introduced in a previous version of
 Kubernetes.  This feature will make it possible for vendors with external CSI drivers to
 implement raw block volumes support in Kubernetes workloads.
 
-CSI block volume support is feature-gated and turned off by default.  To run CSI with
-block volume support enabled, a cluster administrator must enable the feature for each
-Kubernetes component using the following feature gate flags:
+CSI block volume support is enabled by default, but can be disabled using the
```
Comment (msau42) [inline content/en/docs/concepts/storage/volumes.md]: We usually don't talk about disabling features, only how to enable them.  I think we can just mention here the feature gates that are required to be enabled.

**Pull request merged**